### PR TITLE
Mxchip errors to pipeline

### PR DESCRIPTION
**Pull request by jbobotek**
# Checklist
- [ ] I have read the [contribution guidelines] (https://github.com/Azure/azure-iot-sdk-c/blob/master/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- If this is a modification that impacts the behavior of a public API
  - [ ] I edited the corresponding document in the `devdoc` folder and added or modified requirements.
- I submitted this PR against the correct branch: 
  - [ ] This pull-request is submitted against the `master` branch. 
  - [ ] I have merged the latest `master` branch prior to submission and re-merged as needed after I took any feedback.
  - [ ] I have squashed my changes into one with a clear description of the change.

# Reference/Link to the issue solved with this PR (if any)

# Description of the problem

# Description of the solution

### CODE REVIEW

File: testtools/UART_interface/input.sh [hunk 1]
```diff
@@ -5,4 +5,4 @@
 set -e
 
 # basic setup of input script for python
-echo -e "help\r\nset_az_iothub $IOTHUB_CONNECTION_STRING\r\n$SERIAL_TASK\r\n\r\nversion" | cat > input.txt
+echo -e "help\r\nset_az_iothub $IOTHUB_CONNECTION_STRING\r\n\r\n$SERIAL_TASK\r\n\r\nversion" | cat > input.txt
```
Comment (massand) [inline testtools/UART_interface/input.sh]: >\n\r\n$ [](start = 58, length = 7)

Is this change needed?

Comment (jbobotek) [reply testtools/UART_interface/input.sh]: it is not, but I thought I'd give a little more process time for the chip after setting the hub string.

File: build/.mxchip-ci.yml [hunk 2]
```diff
@@ -63,7 +63,7 @@ jobs:
       # send_telemetry [args]
 
       SERIAL_TASK: $(SERIAL_TASK)
-      IOTHUB_CONNECTION_STRING: $(IOTHUB-CONNECTION-STRING)
+      IOTHUB_CONNECTION_STRING: $(MXCHIP_NIGHTLY_DEVICE_STRING)
```
Comment (bikamani) [inline build/.mxchip-ci.yml]: Where is this value coming from?

Comment (jbobotek) [reply build/.mxchip-ci.yml]: devicelab-c-sdk vault/library
